lib/faktory_worker/connection_manager.ex

```diff
@@ -43,6 +43,8 @@ defmodule FaktoryWorker.ConnectionManager do
       # Handle errors from Faktory that should not be tried again, such as
       # unique jobs.
       {{:error, "Halt: " <> reason = error}, state} ->
+        # when we add support for the -NOTUNIQUE response this logging
+        # should be moved into the WorkerLogger module
         log_error(error, command)
```

Comment:
This is the only existing logging that I have left in place since this will change once the next version of faktory is released including the `-NOTUNIQUE` response.